rts/src/main/java/eta/runtime/Runtime.java

```diff
@@ -157,8 +162,21 @@ public static boolean debugMemoryManager() {
         return debugMemoryManager;
     }
 
+    public static void init_params() {
+        RuntimeOptions rto = new RuntimeOptions(RTS_PROPERTIES_PATH);
+        // Initialize parameters explicitly
+        maxWorkerCapabilities = rto.getInt("maxWorkerCapabilities", getNumberOfProcessors());
+        maxGlobalSparks = rto.getInt("maxGlobalSparks", 4096);
+        minTSOIdleTime = rto.getInt("minTSOIdleTime", 20);
+        maxTSOBlockTime = rto.getInt("maxTSOBlockTime", 1);
+        minWorkerCapabilityIdleTime = rto.getInt("minWorkerCapabilityIdleTime", 1000);
+        gcOnWeakPtrFinalization = rto.getBoolean("gcOnWeakPtrFinalization", false);
+        maxLocalSparks = rto.getInt("maxLocalSparks", 4096);
+    }
```

Comment:
All the keys should be prefixed with `eta.rts.` - this is a change you can do inside of `getValue` probably. 

Moreover, please define static constants for all the keys (i.e. `public static final MAX_WORKER_CAPABILITIES_PROPERTY = "maxWorkerCapabilities";`, etc.)